### ISSUE TITLE
Change all examples to use zio.App instead of explicit defaultRunTime

### ISSUE THREAD

**Issue by afsalthaj** [zio/zio-config#60]

**Comment by pgabara** [comment]
I'd like to take this one

**Comment by afsalthaj** [comment]
It’s not a priority at the moment but yes assign to yourself

**Comment by afsalthaj** [comment]
We will change examples after the core code is in final shape

**Comment by afsalthaj** [comment]
Reopen this issue, and good to go once #78 is merged
